provider/block_devices_plug.py

```diff
@@ -234,7 +234,7 @@ def _create_devices(self, images):
             img_params = self.vm.params.object_params(img)
             devices_created = getattr(
                 self.vm.devices, '%s_define_by_params' % self._dev_type['name'])(
-                img, img_params, self._dev_type['media'])
+                img, img_params, self._dev_type['media'], pci_bus=pci_bus)
```

Comment:
I am afraid this would fail those device generator that has no parameter `pci_bus`

Reply:
Hi, @vivianQizhu 
This module just care about the block devices which have the parameter `pci_bus`.
like:
```
    def cdroms_define_by_params(self, name, image_params, media=None,
                                index=None, image_boot=None,
                                image_bootindex=None,
                                pci_bus={"aobject": "pci.0"}):
```
```
    def images_define_by_params(self, name, image_params, media=None,
                                index=None, image_boot=None,
                                image_bootindex=None,
                                pci_bus={"aobject": "pci.0"}):

```
Thanks.

Reply:
@YongxueHong Ah okay, then it would be fine.